tests/test_package_client.py

```diff
@@ -116,15 +118,23 @@ def check_systemd_start_full_cycle(self, ssh_connection):
         assert "Loaded configuration file" in result.stdout, result.stdout
         assert "No dual rootfs configuration present" in result.stdout, result.stdout
 
-        assert any(
-            [
-                expected_output in result.stdout
-                for expected_output in [
-                    "Reauthorization failed with error: transient error: authorization request failed",
-                    "Authorize failed: transient error: authorization request failed",
+        while time.time() - start_time < timeout:
```

Comment:
just wondering if `start_time = time.time()` should be set here?

Reply:
No. The timeout for the whole cycle are these 4 minutes that we set at the beginning.